timeutil/schedule.go

```diff
@@ -28,49 +29,57 @@ import (
 	"time"
 )
 
```

Comment:
I'm not sure if you agree but in reading this it helped me to put the 
vocabulary/context in my head:

```
// This module contains a number of types, here's a quick guide:
//
// Clock - tuple(hour, minute)
//   Just the time of day with minute accuracy
// ClockSpan - tuple(start, end, split, spread)
//   ClockSpan takes two Clock values to offer a `start`-`stop` range
//   and additionally allows to partition it into up to `split` chunks
//   that can be randomized if `spread` is on.
// Week - tuple(weekeday, week-of-month)
//   Week combines the actual weekday, like Monday, with an optional
//   number of the week in a month, e.g. last week of month.
// WeekSpan - tuple(start, end), pair of Week types
//   WeekSpan is like a range inside some month, but not across months.
// Schedule - tuple(week-spans, clock-spans)
//   Schedule ties everything together allowing to set a number of
//   week-spans and clock-spans, applying a NxM combination of all of them.
```

Reply:
That looks like documenting every type again.

Note that godoc works locally too, and can produce textual output into the terminal.